ansible/files/bootstrap/telemetry.sh

```diff
@@ -62,6 +26,14 @@ while [ $# -gt 0 ]; do
             shift
             TIMEOUT=$1
             ;;
+        --email)
+            shift
+            EMAIL=$1
+            ;;
+        --efs-dns)
+            shift
+            EFS_DNS=$1
```

Comment:
@jezdez yes, we'll have to send in the efs-dns. Prod is currently 'fs-d0c30f79.efs.us-west-2.amazonaws.com', dev is 'fs-616ca0c8.efs.us-west-2.amazonaws.com', though I'm not sure how we'd want to differentiate using those.